yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/identity/IdentityService.java

```diff
@@ -0,0 +1,98 @@
+package com.yoti.api.client.spi.remote.call.identity;
+
+import static com.yoti.api.client.spi.remote.call.HttpMethod.HTTP_POST;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.AUTH_ID_HEADER;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.CONTENT_TYPE;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.CONTENT_TYPE_JSON;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.DEFAULT_IDENTITY_URL;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.PROPERTY_YOTI_API_URL;
+import static com.yoti.api.client.spi.remote.util.Validation.notNull;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.security.KeyPair;
+
+import com.yoti.api.client.identity.SessionCreation;
+import com.yoti.api.client.identity.SessionResult;
+import com.yoti.api.client.spi.remote.call.ResourceException;
+import com.yoti.api.client.spi.remote.call.SignedRequest;
+import com.yoti.api.client.spi.remote.call.SignedRequestBuilderFactory;
+import com.yoti.api.client.spi.remote.call.factory.UnsignedPathFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IdentityService {
+
+    public static IdentityService newInstance() {
+        return new IdentityService(
+                new ObjectMapper(),
+                new UnsignedPathFactory(),
+                new SignedRequestBuilderFactory()
+        );
+    }
+
+    static final Logger LOG = LoggerFactory.getLogger(IdentityService.class);
+
+    private final ObjectMapper mapper;
+    private final UnsignedPathFactory pathFactory;
+    private final SignedRequestBuilderFactory requestBuilderFactory;
+
+    private final String apiUrl;
+
+    public IdentityService(
+            ObjectMapper mapper,
```

Comment:
Not sure it's a good idea to let a client pass in an `ObjectMapper`... we should be in full control of how the (de)serialization is done. If we let a client control this then they could (by accident) "break" (de)serialization.

I don't see a reason why this should be exposed.